pypackages/libsql-client/libsql_client/http_driver.py

```diff
@@ -20,7 +20,7 @@ async def batch(self, stmts: List[_RawStmt]) -> List[ResultSet]:
             "statements": [_encode_stmt(stmt) for stmt in stmts],
         }
 
-        async with await self._session.post(self._url, json=req_body) as resp:
+        async with await self._session.post(self._url + "/queries", json=req_body) as resp:
```

Comment:
It would be more robust to use `urllib.parse.urljoin(self._url, "/queries")` to construct the URL.

Reply:
(It will require `import urllib.parse` at the top of the file.)

Reply:
thanks